### ISSUE TITLE
Drawer – Target Size (Minimum) (AA) and Resize Dialog

### ISSUE THREAD

**Issue by gouttierre** [microsoft/fluentui#33648]
### Problem
- Resizable option. This may fail even if an alternative mandated by Dragging Movements is added, if the target size of the alternative is also too narrow.

----

### Current State

- _[Components / Drawer - Docs ⋅ Storybook](https://react.fluentui.dev/?path=/docs/components-drawer--docs#size)_

----
### Change Needed

- We need to add a transparent pseudo-element with a width of 24px to ensure the touch target size for the draggable element, as well as the focus logic and popup for resizing.
- _[Refer to this Figma branch for visuals](https://www.figma.com/design/V2sDk36xZfp8tFhb53DfsT/branch/7AKkln0Kf21JfyDZb84P5M/Drawer-(Overlay-%26-Inline)?m=auto&node-id=10602-24078&t=gTAK3wWjuy3KweoV-1)_
- There's no min-width for the drawer, but for the input field it's suggested `240px`. For max a full drawer which goes to the full container width.
